### ISSUE TITLE
OpenLayer plugin issue in Qgis 3.0 Python 3.x

### ISSUE THREAD

**Issue by marquish** [sourcepole/qgis-openlayers-plugin#52]
OpenLayer plugin is not listed under Manage and Install Plugin in Qgis 3.0.1.
Then i downloaded zip file from https://plugins.qgis.org/plugins/openlayers_plugin/.
While using install from zip, it displays the following error.
```


Couldn't load plugin 'openlayers_plugin' due to an error when calling its classFactory() method 

ImportError: cannot import name 'OpenlayersPlugin' 
Traceback (most recent call last):
  File "C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python\qgis\utils.py", line 336, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:/Users/XX/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\openlayers_plugin\__init__.py", line 25, in classFactory
    from openlayers_plugin import OpenlayersPlugin
ImportError: cannot import name 'OpenlayersPlugin'

Python version: 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)] 
QGIS version: 3.0.1-Girona Girona, a86bec25eb 

Python Path:
C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python
C:/Users/XX/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/XX/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.0\bin\python36.zip
C:\PROGRA~1\QGIS3~1.0\apps\Python36\DLLs
C:\PROGRA~1\QGIS3~1.0\apps\Python36\lib
C:\Program Files\QGIS 3.0\bin
C:\PROGRA~1\QGIS3~1.0\apps\Python36
C:\PROGRA~1\QGIS3~1.0\apps\Python36\lib\site-packages
C:/Users/XX/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/XX/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\go2streetview
C:\Users\XX\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\go2streetview\res
C:/Users/XX/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\Video_UAV_Tracker
C:\Users\XX\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins
C:\Users\XX\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins
```

**Comment by pka** [comment]
Port to QGIS 3 is now published as *experimental* [version 2.0.0](http://plugins.qgis.org/plugins/openlayers_plugin/version/2.0.0/). Please test and give feedback in ticket #37 .

**Comment by marquish** [comment]
Thanks, Here i would like to present my experiences.
1. The plugin must be downloaded and then installed using "Install From Zip" option. else install would show warning (version mismatch).
2. The plugin is Experimental, so Experimental option must be enabled from settings.
3. The experimental plugin works !
